---
layout: post
title: Research
cover_url: assets/seg.png
permalink: research
---

This is a list of my current research projects or notes about what I am
interested in. This will be updated periodically.

# Allosteric control in transcriptional regulation
[<i class='icon-web'></i> website](http://rpgroup.caltech.edu/mwc_induction) | [<i class='icon-git'></i> git repository](http://github.com/rpgroup-pboc/mwc_induction) 

Allosteric regulation is found across all domains of life, yet we still lack
simple, predictive theories that directly link the experimentally tunable
parameters of a system to its input-output response. Through use of a
statistical mechanical depiction of the Monod-Wyman-Changeux model of
allostery, one can derive analytical expressions for the fold-change in gene
expression of a transcriptional regulatory circuit in which the transcription
factor is the allosteric regulator. In [Razo-Mejia and Chure, et al.](https://www.cell.com/cell-systems/fulltext/S2405-4712(18)30057-7#.W01_Pgup_pg.twitter), we
put forward such a theory and rigorously test it experimentally in the
context of the most ubiquitous bacterial regulatory architecture - simple
repression.

![]({{site.baseurl}}/assets/fc_surfaces.png)

<div class="caption">

 The surface of available fold-change in gene expression for a promoter regulated by both an activator and repressor across a range of allosteric effector concentrations <i>c</i>. The notation R and A correspond to the copy numbers of repressor and activator, respectively, and Δε describe the DNA-binding strength of each species.

</div>


# Mechanosensation and survival under osmotic shock
[<i class='icon-web'></i>  website](http://rpgroup.caltech.edu/mscl_survival) | [<i class='icon-git'></i> git repository](http://github.com/rpgroup-pboc/mscl_survival) 


Mechanosensitive  (MS)  channels  are  transmembrane  protein  complexes  which  open  and  close  in response to changes in membrane tension as a result of osmotic shock. Despite extensive biophysical characterization, the contribution of these channels to cell survival remains largely unknown. In [Chure, Lee, and Phillips 2018](http://www.rpgroup.caltech.edu/mscl_survival/Chure2018a.pdf)  we use quantitative video microscopy to measure the abundance of a single species of MS channel in single cells followed by their survival after a large osmotic shock.  We observe total death of the population with less than 100 channels per cell and determine that approximately 500 - 700 channels are needed for 80% survival. The number of channels we find to confer nearly full survival is consistent with the counts of the number of channels in wild type cells in several earlier studies.  These results prompt further studies to dissect the contribution of other channel species to survival.


![]({{site.baseurl}}/assets/fig3.png)
<div class="caption">
Experimental approach to measuring survival probability. (A) Layout of a home-made flow cell for subjecting cells to osmotic shock. Cells are attached to a polyethylamine functionalized surface of a glass coverslip within the flow chamber by loading a dilute cell suspension through one of the inlets. (B) The typical experimental procedure. Cells are loaded into a flow chamber as shown in (A) and mounted to the glass coverslip surface. Cells are subjected to a hypo-osmotic shock by flowing hypotonic medium into the flow cell. After shock, the cells are monitored for several hours and surviving cells are identified.
</div>

# Developing Pipelines For Open and Reproducible Research
As biological data becomes bigger and bigger and analysis routines become more computationally sophisticated, the scientific process must be adapted such that research is reproducible. I've committed myself to using the [git version control](https://git-scm.coml) system coupled with the hosting service [GitHub](https://github.com) as a laboratory notebook and data repository. Each project I am involved in has its own repository (linked above) and I have created [Reproducible Research: A Template For Using Git and GitHub As A Scientific Research Tool](https://github.com/gchure/reproducible_research) which sets out the bare bones of how I structure my projects. If you are interested in doing the same, feel free to fork the repository and modify as you see fit. 